www-servers/hinsightd/files/initd-v1.sh

```diff
@@ -0,0 +1,50 @@
+#!/sbin/openrc-run
+
+NAME=$RC_SVCNAME
+RUN_DIR=/var/run/$NAME
+LOG_DIR=/var/log/$NAME
+TMP_DIR=/var/tmp/$NAME
+CWD_DIR=/var/www/localhost
+PID_FILE=$RUN_DIR/$NAME.pid
+CFG_FILE=/etc/hinsightd/main.lua
+LOG_FILE=$LOG_DIR/hindsight.log
+
+RUN_FILE=/usr/sbin/hinsightd
+RUN_USER="hinsightd"
```

Comment:
Please declare those variables in a conf.d file instead of here.

Reply:
you mean how nginx does it ?
command_user=${HINSIGHTD_USER:-hinsightd}